src/backend/launcher.ts

```diff
@@ -305,6 +307,34 @@ function setupEnvVars(gameSettings: GameSettings) {
   return ret
 }
 
+/**
+ * Maps launcher info to environment variables for consumption by wrappers
+ * @param wrapperEnv The info to be added into the environment variables
+ * @returns Environment variables
+ */
+function setupWrapperEnvVars(wrapperEnv: WrapperEnv) {
+  const ret: Record<string, string> = {}
+
+  ret.HEROIC_APP_NAME = wrapperEnv.appName
+
+  switch (wrapperEnv.appRunner) {
+    case 'gog':
+      ret.HEROIC_APP_SOURCE = 'gog'
+      break
+    case 'legendary':
+      ret.HEROIC_APP_SOURCE = 'epic'
+      break
+    case 'nile':
+      ret.HEROIC_APP_SOURCE = 'amazon'
+      break
+    case 'sideload':
+      ret.HEROIC_APP_SOURCE = 'sideload'
+      break
+  }
```

Comment:
Why not ?
```suggestion
 ret.HEROIC_APP_SOURCE = wrappernEnv.appRunner
```

Reply:
I was thinking there could be a case in the future where it may help to distinguish the runner and the source, like if you could choose two different runners for the same store or if one runner supported multiple stores. Not sure if that's a valid/realistic concern given Heroic's design goals.

I added a `HEROIC_APP_RUNNER` env var in e78e680768ad5af6152af9070287a5cf6986722d. Just let me know if you'd prefer to have `HEROIC_APP_SOURCE` removed as well.